msm/src/ffa/constraint.rs

```diff
@@ -1,6 +1,6 @@
 use crate::{
     columns::{Column, ColumnIndexer},
-    expr::MSMExpr,
+    expr::E,
```

Comment:
I'm happy with removing `MSM` from `MSMExpr`, but can we at least have `Expr` instead of just `E`? One can always do `expr::Expr as E` to save up three letters (although I think it decreases readability significantly, similarly to `D` sometimes used alongside `Domain`).